src/Microsoft.Win32.Registry/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+// Registry tests can conflict with each other due to accessing the same keys/values in the registry
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
```

Comment:
I agree with the sentiment, but this is a pragmatic solution in the meantime as we dig ourselves out of test stability debt.